test/ccall.jl

```diff
@@ -220,9 +220,22 @@ threadcall_test_func(x) =
 @test threadcall_test_func(3) == 1
 @test threadcall_test_func(259) == 1
 
-@test 1.9 > @elapsed @sync for i = 1:2
-    @unix_only @async @threadcall(:sleep, Cuint, (Cuint,), 1)
-    @windows_only @async @threadcall(:Sleep, Void, (UInt32,), 1000)
+n = 3
```

Comment:
OK. I'll just wait for the AV run to go through first. Not sure if `GetCurrentThread` which returns a pseudo handle will work as expected.